Documentation/Input/HandTracking.md

```diff
@@ -15,8 +15,10 @@ By default the hand joint prefabs are simple geometric primitives. These can be
 > [!WARNING]
 > Avoid using complex scripts or expensive rendering in joint prefabs, since joint objects are transformed on every frame and can have significant performance cost!
 
-<img src="../../Documentation/Images/InputSimulation/MRTK_Core_Input_Hands_JointVisualizerPrefabs.png" height="350px"  style="display:inline;">
```

Comment:
Should this be added with the same file name, so the old image is replaced? Or should we just delete the old image?

Reply:
I added a new pic with a new name so I can delete the old picture